src/components/Select/styles/Options.js

```diff
@@ -29,7 +29,8 @@ const Options = forwardRef(
       return (
         <OptionsListContainer
           data-placement={placement}
-          {...getMenuProps({ ref, style })}
+          ref={ref}
```

Comment:
Why are you removing Downshift supplied properties? These are a core part of how that library works.

Reply:
In this particular case `getMenuProps` caused an error `getMenuProps() is not a function`

Reply:
@antkoval In that case, we should do something more like:
```
const menuProps = getMenuProps ? getMenuProps({ref, style}) : {};
```

and then spread `menuProps`.

Reply:
@zelitzsi  the problem here is not about wrong `getMenuProps()` function, but in broken mechanism of its spreading. The fix you offer turns Options in empty input and behaves incorrectly

Reply:
Why is it spreading incorrectly? As @a-type said, I think `getMenuProps()` is required for this input to receive everything it needs to from Downshift.

Reply:
@zelitzsi `ref` and `style` fields are present in props, but they can not be spread using the existing code. IDK why

Reply:
It's especially important to send `ref` through `getMenuProps` since Downshift applies its own ref to the element and it must internally combine the refs.

Reply:
this component works incorrectly on localhost, thou on env it's ok. Revert these changes, but it's still unclear for me why this can happen